<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btn_1, #btn_2, #btn_3, #btn_4, #btn_5, #btn_6, #btn_7, #btn_8, #btn_9, #btn_0, #btn_virgula {
	background-color: gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#btn_somar, #btn_x, #btn_dividir, #btn_menos, #btn_igual {
	background-color: orange;
}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_x">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="shortcut">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_limpar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="shortcut">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="shortcut">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="shortcut">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="shortcut">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="shortcut">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_igual">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="shortcut">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="shortcut">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="shortcut">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_menos">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="shortcut">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_apagar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>โซ</string>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn3 {
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="shortcut">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_virgula">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="shortcut">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_somar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="shortcut">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dividir">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>รท</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
